Formula/g/groff.rb

```diff
@@ -36,7 +37,13 @@ class Groff < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}", "--without-x", "--with-uchardet"
+    # Local config needs to survive upgrades
+    inreplace "Makefile.in", "localfontdir = $(dataprogramdir)",
+              "localfontdir = @sysconfdir@/groff"
+    inreplace "Makefile.in", "localtmacdir = $(dataprogramdir)",
+              "localtmacdir = @sysconfdir@/groff"
```

Comment:
```suggestion
    inreplace "Makefile.in" do |s|
      s.change_make_var! "localfontdir", "@sysconfdir@/groff"
      s.change_make_var! "localtmacdir", "@sysconfdir@/groff"
    end
```

Reply:
@carlocab - nice feature, I'll add it (although there's more to the values, I'll edit...)